Installation of openkruise times out.

action url: https://github.com/spidernet-io/spiderpool/actions/runs/9821775347

```
ready.go:336: [debug] DaemonSet is not ready: kruise-system/kruise-daemon. 0 out of 1 expected pods are ready
ready.go:336: [debug] DaemonSet is not ready: kruise-system/kruise-daemon. 0 out of 1 expected pods are ready
ready.go:336: [debug] DaemonSet is not ready: kruise-system/kruise-daemon. 0 out of 1 expected pods are ready
ready.go:336: [debug] DaemonSet is not ready: kruise-system/kruise-daemon. 0 out of 1 expected pods are ready
ready.go:336: [debug] DaemonSet is not ready: kruise-system/kruise-daemon. 0 out of 1 expected pods are ready
ready.go:336: [debug] DaemonSet is not ready: kruise-system/kruise-daemon. 0 out of 1 expected pods are ready
ready.go:336: [debug] DaemonSet is not ready: kruise-system/kruise-daemon. 0 out of 1 expected pods are ready
ready.go:336: [debug] DaemonSet is not ready: kruise-system/kruise-daemon. 0 out of 1 expected pods are ready
ready.go:336: [debug] DaemonSet is not ready: kruise-system/kruise-daemon. 0 out of 1 expected pods are ready
wait.go:87: [debug] Error received when checking status of resource kruise-webhook-service. Error: 'client rate limiter Wait returned an error: context deadline exceeded', Resource details: 'Resource: "/v1, Resource=services", GroupVersionKind: "/v1, Kind=Service"
Name: "kruise-webhook-service", Namespace: "kruise-system"'
wait.go:94: [debug] Retryable error? true
wait.go:71: [debug] Retrying as current number of retries 0 less than max number of retries 30
Error: INSTALLATION FAILED: context deadline exceeded
helm.go:84: [debug] context deadline exceeded
INSTALLATION FAILED
main.newInstallCmd.func2
	helm.sh/helm/v3/cmd/helm/install.go:158
github.com/spf13/cobra.(*Command).execute
	github.com/spf13/cobra@v1.8.0/command.go:983
github.com/spf13/cobra.(*Command).ExecuteC
	github.com/spf13/cobra@v1.8.0/command.go:1115
github.com/spf13/cobra.(*Command).Execute
	github.com/spf13/cobra@v1.8.0/command.go:1039
main.main
	helm.sh/helm/v3/cmd/helm/helm.go:83
runtime.main
	runtime/proc.go:271
runtime.goexit
	runtime/asm_amd64.s:1695
make[3]: *** [Makefile:204: setup_kurise] Error 1
make[3]: Leaving directory '/home/runner/work/spiderpool/spiderpool/test'
make[2]: *** [Makefile:41: kind-init] Error 2
make[2]: Leaving directory '/home/runner/work/spiderpool/spiderpool/test'
make[1]: *** [Makefile:299: e2e_init] Error 2
make[1]: Leaving directory '/home/runner/work/spiderpool/spiderpool'
make: *** [Makefile:323: e2e_init_underlay] Error 2
```

--- a/test/Makefile
+++ b/test/Makefile
@@ -83,8 +83,8 @@ endif
 		E2E_SPIDERPOOL_ENABLE_SUBNET=${E2E_SPIDERPOOL_ENABLE_SUBNET} \
 		scripts/install-multus.sh $(E2E_CLUSTER_NAME) $(E2E_KUBECONFIG)
 ifeq ($(INSTALL_KRUISE),true)
-	@echo -e "\033[35m [Step 9] Install operator: kurise \033[0m"
-	@ make setup_kurise
+	@echo -e "\033[35m [Step 9] Install operator: kruise \033[0m"
+	@ make setup_kruise
 endif
 ifeq ($(INSTALL_KDOCTOR),true)
 	@echo -e "\033[35m [Step 10] Install kdoctor \033[0m"
@@ -214,10 +214,8 @@ post_installation:
 			done ; \
 		done
 
-
-
-.PHONY: setup_kurise
-setup_kurise:
+.PHONY: setup_kruise
+setup_kruise:
 	@echo "add openkruise charts repository..." ; \
 		[ -z "$(HTTP_PROXY)" ] || export https_proxy=$(HTTP_PROXY) ; \
 			helm repo add openkruise https://openkruise.github.io/charts/ ; \
@@ -226,7 +224,8 @@ setup_kurise:
 				docker pull $${IMAGE} ; \
 				kind load docker-image $${IMAGE} --name $(E2E_CLUSTER_NAME);	\
 			done; \
-			helm install kruise openkruise/kruise --kubeconfig $(E2E_KUBECONFIG) --wait --debug --set manager.image.repository=$(E2E_OPENKRUISE_IMAGE)
+			helm upgrade --install kruise openkruise/kruise --wait --timeout 20m --debug --set manager.image.repository=$(E2E_OPENKRUISE_IMAGE) \
+			    --kubeconfig $(E2E_KUBECONFIG)  || { KIND_CLUSTER_NAME=$(E2E_CLUSTER_NAME) ./scripts/debugEnv.sh $(E2E_KUBECONFIG) "detail"   ; exit 1 ; } ; \
 
 .PHONY: setup_spiderpool
 setup_spiderpool:

--- a/test/scripts/debugEnv.sh
+++ b/test/scripts/debugEnv.sh
@@ -25,7 +25,7 @@ CONTROLLER_POD_LIST=$( kubectl get pods --no-headers --kubeconfig ${E2E_KUBECONF
 AGENT_POD_LIST=$( kubectl get pods --no-headers --kubeconfig ${E2E_KUBECONFIG}  --namespace ${NAMESPACE} --selector app.kubernetes.io/component=spiderpool-agent --output jsonpath={.items[*].metadata.name} )
 KUBEVIRT_HANDLER_POD_LIST=$( kubectl get pods --no-headers --kubeconfig ${E2E_KUBECONFIG}  --namespace kubevirt --selector kubevirt.io=virt-handler --output jsonpath={.items[*].metadata.name} )
 KDOCTOR_POD_LIST=$( kubectl get pods --no-headers --kubeconfig ${E2E_KUBECONFIG}  --namespace ${NAMESPACE} --selector app.kubernetes.io/instance=kdoctor --output jsonpath={.items[*].metadata.name} )
-
+KRUISE_POD_LIST=$( kubectl get pods --no-headers --kubeconfig ${E2E_KUBECONFIG}  --namespace kruise-system --output jsonpath={.items[*].metadata.name} )
 
 [ -z "$CONTROLLER_POD_LIST" ] && echo "error, failed to find any spider controller pod" && exit 1
 [ -z "$AGENT_POD_LIST" ] && echo "error, failed to find any spider agent pod" && exit 1
@@ -266,6 +266,17 @@ elif [ "$TYPE"x == "detail"x ] ; then
       kubectl logs ${POD} -n ${NAMESPACE} --kubeconfig ${E2E_KUBECONFIG}
       echo "--------- kubectl logs ${POD} -n ${NAMESPACE} --previous"
       kubectl logs ${POD} -n ${NAMESPACE} --kubeconfig ${E2E_KUBECONFIG} --previous
+    done
+
+
+    echo ""
+    echo "=============== open kruise logs ============== "
+    for POD in $KRUISE_POD_LIST ; do
+      echo ""
+      echo "--------- kubectl logs ${POD} -n kruise-system "
+      kubectl logs ${POD} -n kruise-system --kubeconfig ${E2E_KUBECONFIG}
+      echo "--------- kubectl logs ${POD} -n kruise-system --previous"
+      kubectl logs ${POD} -n kruise-system --kubeconfig ${E2E_KUBECONFIG} --previous
     done
 
 elif [ "$TYPE"x == "error"x ] ; then

